guides/release/tutorial/ember-data.md

```diff
@@ -3,7 +3,7 @@ As our application grows, we will want to persist our rental data on a server, a
 
 Ember comes with a data management library called [Ember Data](https://github.com/emberjs/data) to help deal with persistent application data.
 
-Ember Data requires you to define the structure of the data you wish to provide to your application by extending [`DS.Model`](https://www.emberjs.com/api/ember-data/release/classes/DS.Model).
+Ember Data requires you to define the structure of the data you wish to provide to your application by extending [`DS.Model`](https://www.emberjs.com/api/ember-data/release/classes/DS.Model).("D" for Data, "S" for structure).
```

Comment:
I believe it comes from "Data Store", but really it does not have any actual meaning nowadays, it is the namespace for Ember Data classes.
This is [going to change soon](https://github.com/ember-learn/guides-source/pull/814/files) and the old-school namespace won't be needed anymore! :)

Reply:
@locks Ricardo! Cool then! And really thanks for the explanation! Should I close the branch right?

Reply:
Yeah, if you don't mind!

Reply:
At all! ;-)